packages/cfd/src/Containers/__tests__/trading-password-manager.spec.js

```diff
@@ -15,6 +15,7 @@ jest.mock('@deriv/shared/src/services/ws-methods', () => ({
     default: 'mockedDefaultExport',
     WS: {
         verifyEmail: jest.fn(() => Promise.resolve()),
+        //abcdf
```

Comment:
why we added this comment 🙏

Reply:
was added for testing purpose have removed it 👍